examples/wasi-http-rust-streaming-outgoing-body/spin.toml

```diff
@@ -12,7 +12,7 @@ component = "wasi-http-async"
 
 [component.wasi-http-async]
 source = "target/wasm32-wasi/release/wasi_http_rust_streaming_outgoing_body.wasm"
-allowed_http_hosts = ["insecure:allow-all"]
+allowed_outbound_hosts = ["https://*:*"]
```

Comment:
Would this still allow HTTP (not HTTPS) for localhost scenarios?

Reply:
I think in the current implementation it would but that seems a bit odd. The most consistent thing would be to require `{http, https}://*:*` (note that the list syntax is not yet implemented), but that is admittedly a bit annoying.